ko.go

```diff
@@ -0,0 +1,49 @@
+// Copyright 2018 Google LLC All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"log"
+	"os"
+
+	"github.com/google/go-containerregistry/pkg/logs"
+	"github.com/google/ko/pkg/commands"
+
+	cranecmd "github.com/google/go-containerregistry/cmd/crane/cmd"
+	"github.com/spf13/cobra"
+)
+
+func main() {
```

Comment:
Is this filename (`ko.go`) a convention for single tool repos? I'd prefer `main.go` I think.

Reply:
the filename dictates what the installed binary will be I think. Let me double check.

Reply:
it worked with main.go. Switched.